assertions/src/main/java/io/camunda/zeebe/process/test/assertions/MessageAssert.java

```diff
@@ -234,4 +237,51 @@ private List<Long> getProcessInstanceKeysForCorrelatedMessageStartEvent() {
         .map(record -> record.getValue().getProcessInstanceKey())
         .collect(Collectors.toList());
   }
+
+  private Map<String, Object> getVariables() {
+    return StreamFilter.processMessageSubscription(recordStream)
```

Comment:
:x: We should get the variables of the message from the message records. 

A process message subscription is only written when a message is correlated to a process instance.

Reply:
I would use `StreamFilter.message(recordStream)` for this?

Reply:
Yes. :+1: